lang/lang_en_nl.txt

```diff
@@ -176,7 +176,7 @@
 
 #MSG_CHECKING_FILE c=17
 "Checking file"
-"\x00"
+"Control. bestand"
```

Comment:
Does this really need abbreviation? seems like "Bestand controle" should be fine and still fit